src/main/resources/application.yml

```diff
@@ -68,6 +68,21 @@ courtfinder:
   api:
     url: "https://courttribunalfinder.service.gov.uk"
 
+fees:
+  api:
+    url: ${FEES_URL}
+    service: "civil money claims"
+    jurisdiction1: "civil"
+    jurisdiction2: "county court"
+
+payments:
+  returnUrl: "${FRONTEND_BASE_URL:}/claim/pay/%s/receiver"
```

Comment:
Can this explicitly be called a pattern or template or some such?

Reply:
👍